### PR TITLE
1012

### PR DESCRIPTION
**Pull request by jheftmann**
I knew Helvetica Neue base was somewhere!

### CODE REVIEW

File: assets/stylesheets/mixins/_style.scss [hunk 1]
```diff
@@ -15,3 +15,11 @@
   overflow-scrolling: touch;
   -webkit-overflow-scrolling: touch;
 }
+
+@mixin circle {
+  border-radius: 999999px;
+}
+
+@mixin rounded($rounded) {
```
Comment (micdijkstra) [inline assets/stylesheets/mixins/_style.scss]: It's less characters to write `border-radius: 10px` than `@include rounded(10px)`

Comment (jheftmann) [inline assets/stylesheets/mixins/_style.scss]: I guess if it auto-prefixes then yes, true.

unless you think `.circle` is handy

Comment (micdijkstra) [inline assets/stylesheets/mixins/_style.scss]: I think we can make circle a utility class.

File: assets/stylesheets/_typography.scss [hunk 2]
```diff
@@ -87,6 +87,20 @@
   text-transform: none;
 }
 
+h1, h2, h3, h4, h5, h6 {
+  margin: 0;
+  padding: 0;
+  line-height: 1.0;
```
Comment (micdijkstra) [inline assets/stylesheets/_typography.scss]: Nitpicking here, but I'll consolidate this and then just have the `p` below with the one different attribute. I also list all declarations in alphabetical order.

**Pull request merged**